Source/WebCore/page/cocoa/WebTextIndicatorLayer.h

```diff
@@ -40,7 +40,12 @@ WEBCORE_EXPORT @interface WebTextIndicatorLayer : CALayer {
     BOOL _fadingOut;
 }
 
-- (instancetype)initWithFrame:(CGRect)frame textIndicator:(WebCore::TextIndicator&)textIndicator margin:(CGSize)margin offset:(CGPoint)offset;
+enum UpdatingIndicator : BOOL { No, Yes };
```

Comment:
I don't think we need an enum for this, since it's only used in Obj-C the method name already provides the description: `updatingIndicator:`.

Reply:
@smfr likes them to be there all the time.

Reply:
In case we _do_ decide to keep this, it should be `enum class UpdatingIndicator : bool { No, Yes };`, preferably inside of a `namespace WebKit` scope (so it won't collide with any other global symbols from the SDK).

Reply:
I strongly agree with Aditya too

Reply:
@aprotyas added the class in another comment :) I'll add namespace WebKit as well

Reply:
@smfr really?? 😛